### DISCUSSION TITLE
https://gnpy.app/   What are the corresponding parameters “Number of paths”

### DISCUSSION THREAD

**Discussion by winsli** [Telecominfraproject/oopt-gnpy#517] [General]
![image](https://github.com/user-attachments/assets/e33b4458-7ee3-4b08-a48f-91bb2fb3e53b)

**Comment by ojnas** [comment]
I'm not sure what you are asking. It is the number of alternative paths to be computed and simulated. If you are asking about a corresponding parameter in core GNPy, it's not implemented. There are several features in gnpy.app that are not included in core GNPy. If I remember correctly, this specific feature was proposed a long time ago but never got anywhere.

**Reply by winsli** [reply]
Thank you.
